src/plugins/pluginMFManifest.ts

```diff
@@ -188,41 +225,55 @@ const Manifest = (): Plugin[] => {
           })
         );
 
-        // 遍历打包生成的每个文件
         for (const [fileName, fileData] of Object.entries(bundle)) {
           if (fileData.type === 'chunk') {
-            // 遍历该文件的所有模块
             for (const modulePath of Object.keys(fileData.modules)) {
               const sharedKey = fileToShareKey[modulePath];
               if (sharedKey) {
-                if (!filesContainingModules[sharedKey]) {
-                  filesContainingModules[sharedKey] = {
-                    sync: [],
-                    async: [],
-                  };
+                // Track the JS chunk
+                trackAsset(filesMap, sharedKey, fileName, false, 'js');
```

Comment:
It looks great to me 
Can we create a method to reduce the code duplication here?
Can you add few tests please?

Reply:
I can try to abstract things further. As for tests, I have no idea how I would begin to go about going that given the complexity of the plugin as is. Additional from what I can see, the tests that exists are very very crude. in `e2e`